wpeverest/everest-forms
# Fix- Empty Values in Entries for (Select, Checkbox, Radio) Fields when WordPress Site is in Another Language.

### All Submissions:






### Changes proposed in this Pull Request:
Previously, when the WordPress site was set to another language and a user dragged a field in the form builder, the values for the (select, checkbox, radio) fields would appear empty in entries. This pull request addresses and resolves this issue

### How to test the changes in this Pull Request:

1. Drag a (select, checkbox, radio) field onto the form
2. Save the form changes and preview the form.
3. Submit the form with values for the (select, checkbox, radio) fields.
4. Check the entries in the form entries section to ensure that the values for the (select, checkbox, radio) fields are not empty.

### Types of changes:




### Other information:




### Changelog entry

> Fix- Empty Values in Entries for (Select, Checkbox, Radio) Fields when WordPress Site is in Another Language.
